lua/autorun/server/nadmod_pp.lua

```diff
@@ -80,6 +80,41 @@ function NADMOD.IsPPAdmin(ply)
 	end
 end
 
+function NADMOD.SendPropOwners(props, ply)
+	net.Start("nadmod_propowners")
+		local seti = 1
+		local set = {}
+		local count = 0
+		for k,v in pairs(props) do
+			if not set[v] then
+				set[v] = seti
+				set[seti] = v
+				seti = seti + 1
```

Comment:
These variable names are awful haha, but this seems good! I had an earlier comment about optimizing this, and before I posted it, you'd pushed this optimized version :)